40.meta-mender/docs.md

```diff
@@ -1217,6 +1217,67 @@ New changes in meta-mender since dunfell-v2022.09:
       ```
 
 
+## meta-mender (dunfell-v2024.06)
+
+_Released: 06.14.2024_
+
+### Changelogs
+
+#### meta-mender (dunfell-v2024.06)
+
+New changes in meta-mender since dunfell-v2023.03:
+
+##### Bug Fixes
+
+* mender-configure: Don't break if libdir is set to `lib64`.
+* pass prefix and systemd_unitdir to mender-configure installation
+* add coreutils as a runtime dependency for Mender monitor
+* boot-partition-devicetree use proper SPDX License GPL-2.0-only
+* Replace gpt partition type 8300 with full UUID
+* wic-tools does not need grub when using uboot
+* mv image files instead of install, to avoid corrupting the image.
+* Fix the install path for the D-Bus interface files shipped in
+  `mender-client-dev` package. It was `$(datadir)/dbus-1/interface` but it
+  should be `interfaces` instead.
+* remove "mender.bmap" IMAGE_FSTYPE
+* Remove `has_journal` flag from mkfs.ext4 when
+  `read-only-rootfs` flag is enabled, which causes issues in certain
+  configurations. Specifically it solves these problems:
+  * Checksum is not stable which can cause issues with binary delta
+    updates. This is an in-memory effect only, the physical checksum on
+    disk is unaffected.
+  * Mender-snapshot can generate corrupted rootfs dumps, for the same
+    reason as the previous point.
+* Fix typo in fstab options for extra partitions.
+* Correctly handle `mender-partuuid` feature for `dunfell`
+
+##### Features
+
+* Make the build succeed if gateway examples aren't present
```

Comment:
@kacf This one I also think it should be a bugfix, right?

Reply:
Yes, I agree.